internal/utils/map.go

```diff
@@ -14,34 +13,19 @@ func InvertStrIntMap(mapToInvert map[string]int) map[int]string {
 	return newMap
 }
 
+func IntSliceToMap(slice []int) map[int]bool {
```

Comment:
А зачем этот метод публичный? Его будет кто-то ещё вызывать?

Ну и всё-таки
```suggestion
func intSliceToMap(slice []int) map[int]struct{} {
```